requirements-dev.txt

```diff
@@ -17,11 +19,13 @@ wandb
 mlflow
 neptune-client
 tensorboard
+torchvision
 pynvml
 clearml
 scikit-image
 py-rouge
 nltk
+pytorch_fid
```

Comment:
```suggestion
```

Reply:
We are using this module for comparing our implementation. So we need it.

Reply:
You have it twice. See above.